src/typography/p5.Font.js

```diff
@@ -76,6 +76,10 @@ p5.Font = class {
   // Check cache for existing bounds. Take into consideration the text alignment
   // settings. Default alignment should match opentype's origin: left-aligned &
   // alphabetic baseline.
+    if (typeof str !== 'string') {
+      console.error('Error: Text parameter must be a string.');
+      return null;
+    }
```

Comment:
We don't usually handle incorrect parameter types this way as the FES is usually the one to take care of it. In terms of logic it should either do type coersion or return sensible default values, `null` is probably not useful in this case and can cause a property access error on the user's end.